app/controllers/mccloud/CommonMcCloud.scala

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.mccloud
+
+import models.ChargeType
+import models.ChargeType._
+import play.api.mvc.Result
+import play.api.mvc.Results.Redirect
+import uk.gov.hmrc.viewmodels.Text.Message
+
+import scala.concurrent.Future
+
+trait CommonMcCloud {
+  def ordinal(index: Option[Int]): Option[Message] = {
+    index match {
+      case Some(i) if i > 0 && i < 5 => Some(Message(s"mccloud.scheme.ref$i"))
```

Comment:
Does this count and work for the 5th? Logic looks like it counts for set {1, 2, 3, 4}

Reply:
Count starts at zero, so index 4 is actually the 5th